src/Microsoft.ML.Transforms/CategoricalCatalog.cs

```diff
@@ -96,11 +100,13 @@ public static class CategoricalCatalog
             => new OneHotEncodingEstimator(CatalogUtils.GetEnvironment(catalog), columns, keyData);
 
         /// <summary>
-        /// Convert a text column into hash-based one-hot encoded vector.
+        /// Create a <see cref="OneHotHashEncodingEstimator"/>, which converts a text column specified by <paramref name="inputColumnName"/> into a hash-based one-hot encoded vector column named <paramref name="outputColumnName"/>.
         /// </summary>
         /// <param name="catalog">The transform catalog</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be a vector of floats for <paramref name="outputKind"/> Bag, Indicator, and Binary. For <paramref name="outputKind"/> Key, the data type will be a key in the case of a singleton input column or a vector of keys in the case of a vector input column.</param>
```

Comment:
>floats [](start = 56, length = 6)

 • Use System.Single instead of 'float'. 'float' is a C# keywork, not a .NET type, and F# uses different terminology.
  ○ <see cref=System.Single/> in xml
  ○ <xref:System.Single> in markdown 
 • Same as above for 'double'